dart/gui/CMakeLists.txt

```diff
@@ -2,19 +2,20 @@
 file(GLOB srcs "*.cpp")
 file(GLOB hdrs "*.h")
 
+include_directories(SYSTEM ${OPENGL_INCLUDE_DIR})
 include_directories(SYSTEM ${GLUT_INCLUDE_DIR})
 
 # Add subdirectories
 if(HAVE_OPENSCENEGRAPH)
   add_subdirectory(osg)
 endif(HAVE_OPENSCENEGRAPH)
 
-set(dart_gui_hdrs ${dart_gui_hdrs} ${hdrs})
-set(dart_gui_srcs ${dart_gui_srcs} ${srcs})
+set(dart_gui_hdrs ${dart_gui_hdrs} ${hdrs} ${dart_renderer_hdrs})
+set(dart_gui_srcs ${dart_gui_srcs} ${srcs} ${dart_renderer_srcs})
 
 # Library
 dart_add_library(dart-gui ${srcs} ${hdrs})
```

Comment:
You're correct. This was resolved as `renderer` was merged into `gui` [as planned](https://github.com/dartsim/dart/issues/589).